src/qutip_qip/circuit.py

```diff
@@ -569,8 +566,10 @@ def _gate_PHASEGATE(self, gate, temp_resolved):
     def _gate_NOTIMPLEMENTED(self, gate, temp_resolved):
         raise NotImplementedError("Cannot be resolved in this basis")
 
-    _gate_PHASEGATE = _gate_BERKELEY = _gate_SWAPalpha = _gate_NOTIMPLEMENTED
-    _gate_SQRTSWAP = _gate_SQRTISWAP = _gate_NOTIMPLEMENTED
+    _gate_BERKELEY = _gate_NOTIMPLEMENTED
+    _gate_SWAPalpha = _gate_NOTIMPLEMENTED
+    _gate_SQRTSWAP = _gate_NOTIMPLEMENTED
+    _gate_SQRTISWAP = _gate_NOTIMPLEMENTED
```

Comment:
Interesting that `PHASEGATE` was marked as not implemented...

Out of curiosity, is  there any technical reason to choose 
```
    _gate_BERKELEY = _gate_NOTIMPLEMENTED
    _gate_SWAPalpha = _gate_NOTIMPLEMENTED
    _gate_SQRTSWAP = _gate_NOTIMPLEMENTED
    _gate_SQRTISWAP = _gate_NOTIMPLEMENTED
```
rather than
```
    _gate_PHASEGATE = _gate_BERKELEY = _gate_SWAPalpha = _gate_NOTIMPLEMENTED
    _gate_SQRTSWAP = _gate_SQRTISWAP = _gate_NOTIMPLEMENTED
```
?

Reply:
I chose it for a human reason, not a technical reason. To the Python runtime these are largely the same, but as a human I expect to see attribute and function names on the left and on separate lines, so the longer way makes it much clearer (to me anyway) that four `_gate_XXX` functions are not implemented rather than two.